### PR TITLE
go update and ibm cve

### PR DESCRIPTION
**Pull request by roneymathew**
Signed-off-by: Roney Mathew Thomas <Roney.Thomas1@ibm.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ __main__
 .env
 config.json
 webhook
+vendor
```
Comment (prb112) [inline .gitignore:23]: Hey Roney, seems antithetical to common practices, best to include vendoring, my ten cents, thank you for reviewing. Paul

**Comment by hughhuangzh** [comment]
Hi @roneymathew, can you point which CVE fixed and the CVE link for this PR?

**Comment by roneymathew** [comment]
#https://github.com/advisories/GHSA-45x7-px36-x8w8

**Comment by hughhuangzh** [comment]
@roneymathew thanks, this pr is not enough, only fix one, we have two alerts.  package upgrade is simple, but we need make sure the new packages working as expected, do you build a image and test it in your local?

**Comment by roneymathew** [comment]
Yes i built and created the certificate all working fine

**Review by hughhuangzh — APPROVED** [review]

**Pull request merged**